<?xml version="1.0" encoding="UTF-8"?>
<root BTCPP_format="4">
  <BehaviorTree ID="MainBT">
    <Sequence>
      <Sequence>
        <generate_route x="0.25"
                        y="2.0"/>
      </Sequence>
    </Sequence>
  </BehaviorTree>

  <!-- Description of Node Models (used by Groot) -->
  <TreeNodesModel>
    <Action ID="generate_route"
            editable="true">
      <input_port name="x">TODO</input_port>
      <input_port name="y">TODO</input_port>
    </Action>
  </TreeNodesModel>

</root>
